hack/lib/util.sh

```diff
@@ -504,10 +504,22 @@ kube::util::has_changes_against_upstream_branch() {
   local -r git_branch=$1
   local -r pattern=$2
   local -r not_pattern=${3:-totallyimpossiblepattern}
+  
+  # check prow-aware pattern first
+  if [[ ! -z PULL_BASE_SHA ]]; then
```

Comment:
I think you want this to be `${PULL_BASE_SHA:-}`?

Reply:
also `! -z` can also be written as `-n`.

Reply:
bear my bash :joy: